foreign_cc/configure.bzl

```diff
@@ -207,6 +223,13 @@ def _attrs():
             mandatory = False,
             default = ["", "install"],
         ),
+        "two_part_install": attr.bool(
+            doc = (
+                "Set to True if the configure make script supports two phase install via DESTDIR. (See https://www.gnu.org/software/automake/manual/automake.html#Two_002dPart-Install). Using this option will result in more reproducible builds as --prefix will be set to a consistent value at configure time."
+            ),
+            mandatory = False,
+            default = False,
```

Comment:
I would like to be able to set this as the default behaviour as this will produce more hermetic builds but it is a breaking change